libs/labelbox/src/labelbox/schema/send_to_annotate_params.py

```diff
@@ -3,14 +3,16 @@
 from typing import Optional, Dict
 
 from labelbox.schema.conflict_resolution_strategy import ConflictResolutionStrategy
+from pydantic import model_validator, ConfigDict
+from pydantic.main import BaseModel
 
 if sys.version_info >= (3, 8):
     from typing import TypedDict
 else:
     from typing_extensions import TypedDict
 
 
-class SendToAnnotateFromCatalogParams(TypedDict):
+class SendToAnnotateFromCatalogParams(BaseModel):
```

Comment:
need to inherit from `pydantic_compat.BaseModel` to support compatibility with with pydantic v1 and v2

Reply:
Done!